tests/test_config.py

```diff
@@ -764,6 +764,43 @@ def test_get_audience__invalid_id(self):
 
     self.assertIsNone(self.project_config.get_audience('42'))
 
+  def test_get_audience__retrieves_audiences_first_from_typedAudiences_before_audiences(self):
```

Comment:
Nit: can we rename this to something like `test_get_audience__prefers_typedAudiences_over_audiences`? To me, the current name suggests that the SDK **does** eventually look at `audiences` even if something was found in `typedAudiences`